CSS/Monokai.sublime-color-scheme

```diff
@@ -0,0 +1,10 @@
+{
+	"rules": [
+		{
+			"name": "Numeric Units",
+			"scope": "constant.numeric.suffix.css",
+			"foreground": "var(red2)",
+			"font_style": ""
```

Comment:
Should it specify a font style?

Reply:
Copied it from forum and added name. Would remove font_style if one was set by `constant.numeric`. Maybe not needed?

Reply:
The empty string is required here to reset the italic font style which would otherwise be applied due to the `storage.type` forward fill. (Assuming the old style from ST3 should exactly be restored)